src/taco-cli/cli/kit.ts

```diff
@@ -436,7 +436,7 @@ class Kit extends commands.TacoCommandBase {
         assert(componentType === ProjectComponentType.Platform || componentType === ProjectComponentType.Plugin);
         var componentUpdates: IDictionary<string> = {};
         return kitHelper.getKitInfo(kitId).then(function (kitInfo: TacoKits.IKitInfo): Q.Promise<any> {
-            var componentOverrides = (componentType === ProjectComponentType.Platform) ? kitInfo.platforms : kitInfo.plugins;
+            var componentOverrides: any = (componentType === ProjectComponentType.Platform) ? kitInfo.platforms : kitInfo.plugins;
```

Comment:
Proper type of this variable is TacoKits.IPluginOverrideMetadata | TacoKits.IPlatformOverrideMetadata